ciao-storage/ceph.go

```diff
@@ -42,7 +42,8 @@ func (d CephDriver) CreateBlockDevice(imagePath *string, size int) (BlockDevice,
 	// Currently the kernel rdb client only supports layering but in the future more feaures
 	// should be added as they are enabled in the kernel.
 	if imagePath != nil {
-		cmd = exec.Command("qemu-img", "convert", *imagePath, ID)
+		rbdStr := fmt.Sprintf("rbd:rbd/%s:id=%s", ID, d.ID)
+		cmd = exec.Command("qemu-img", "convert", "-f", "qcow2", "-O", "rbd", *imagePath, rbdStr)
```

Comment:
I'm sure i'm starting to sound a like a stuck record. But I think this would be best rolled into previous patches.